### DISCUSSION TITLE
Is so-test documented?

### DISCUSSION THREAD

**Discussion by ToontjeM** [Security-Onion-Solutions/securityonion#13980] [Unsupported Versions] [answered]
So we know what comes from so-test and what comes from the network itself? I want to understand "threats" that are installed by so-test and what is really happening on my network.
Alternative is to clean the database and start over so my data is clean.

**Comment by cm-ops** [comment, marked as answer]
If you have run so-test and the data is in Elasticsearch with other data, I would remove the indices and start fresh.

**Reply by ToontjeM** [reply]
Yeah, i just did a clean install. Thanks!
